### PR TITLE
Riak Shell help for TS 1.5

### PR DESCRIPTION
**Pull request by andytill**

### CODE REVIEW

File: src/help.erl [hunk 1]
```diff
@@ -169,16 +171,24 @@ help(select) ->
         "* STDEV and STDDEV_SAMP~n"
         "* STDDEVPOP~n"
         "~n"
-        "You can also decorate SELECT statements with ORDER BY, GROUP BY and LIMIT~n"
+        "You can also decorate SELECT statements with ORDER BY, GROUP BY, LIMIT and OFFSET~n"
         "~n"
         "For more details please go to http://docs.basho.com/riak/ts~n";
 help(show) ->
-    "You can use the SHOW table to list all Time Series tables you have created."
+    "You can use the SHOW table to list a Time Series tables you have created."
```
Comment (hazen) [inline src/help.erl:178]: Should still be `all`

File: src/help.erl [hunk 2]
```diff
@@ -169,16 +171,24 @@ help(select) ->
...
         "An example of the format is shown below:~n"
         "~n"
         "(1)>SHOW mytable;~n"
         "~n"
         "For more details please go to http://docs.basho.com/riak/ts~n";
+help([show,create,table]) ->
+    "You can use SHOW CREATE TABLE to list the SQL statement that was used "
+    "to create a Time Series table."
```
Comment (hazen) [inline src/help.erl:186]: Needs new-lines.  Prints as a single line.

**Review by hazen — CHANGES_REQUESTED** [review]
Need to clean up text slightly, then ship it.

**Comment by gordonguthrie** [comment]
Superceded by PR 72

File: src/help.erl [hunk 3]
```diff
@@ -169,16 +171,24 @@ help(select) ->
...
+        "An example of the format is shown below:~n"
+        "~n"
+        "(1)>SHOW CREATE TBABLE mytable;~n"
```
Comment (ph07) [inline src/help.erl:189]: Spelling mistake: TBABLE --> TABLE

**Pull request closed without merging**